apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Tests/UrlSignerV4TestData.json

```diff
@@ -0,0 +1,21 @@
+﻿// Assumed constant for all tests:
+// - email: test-iam-credentials@dummy-project-id.iam.gserviceaccount.com
+// - project: dummy-project-id
+// - algorithm: GOOG4-RSA-SHA256
+[
+    {
+        "description":  "Simple GET",
+        "bucket": "test-bucket",
+        "object": "test-object",
+        "method": "GET",
+        "expiration": 10,
+        "headers": {
+            "host": "storage.googleapis.com"
+        },
+        "timestamp": "20190201T090000Z",
+        "datestamp": "20190201",
+        "region": "auto",
+        "credentialScope": "20190201/auto/storage/goog4_request",
+        "expectedUrl": "https://storage.googleapis.com/test-bucket/test-object?x-goog-algorithm=GOOG4-RSA-SHA256&x-goog-credential=test-iam-credentials%40dummy-project-id.iam.gserviceaccount.com%2F20190201%2Fauto%2Fstorage%2Fgoog4_request&x-goog-date=20190201T090000Z&x-goog-expires=10&x-goog-signedheaders=host&x-goog-signature=95e6a13d43a1d1962e667f17397f2b80ac9bdd1669210d5e08e0135df9dff4e56113485dbe429ca2266487b9d1796ebdee2d7cf682a6ef3bb9fbb4c351686fba90d7b621cf1c4eb1fdf126460dd25fa0837dfdde0a9fd98662ce60844c458448fb2b352c203d9969cb74efa4bdb742287744a4f2308afa4af0e0773f55e32e92973619249214b97283b2daa14195244444e33f938138d1e5f561088ce8011f4986dda33a556412594db7c12fc40e1ff3f1bedeb7a42f5bcda0b9567f17f65855f65071fabb88ea12371877f3f77f10e1466fff6ff6973b74a933322ff0949ce357e20abe96c3dd5cfab42c9c83e740a4d32b9e11e146f0eb3404d2e975896f74"
```

Comment:
Query parameter name casing discussion should be reopened. I comment on the bug and in the document related to this PR internally.